packages/expo-dev-menu/ios/DevMenuPackagerConnectionHandler.swift

```diff
@@ -10,7 +10,7 @@ class DevMenuPackagerConnectionHandler {
   }
   
   func setup() {
-#if RCT_DEV
+#if DEBUG
```

Comment:
maybe we should add a comment here explaining this change? i don't anticipate the definition of RCT_DEV diverging from DEBUG, but it could be a problem if it does, since it determines whether RCTPackagerConnection is defined.